integration/tests/cook/test_cli.py

```diff
@@ -1748,7 +1748,28 @@ def test_usage(self):
         self.assertEqual(0, cp.returncode, cp.stderr)
         self.assertEqual(3, len(uuids))
         uuid_7, uuid_8, uuid_9 = uuids
+
         all_uuids = [uuid_1, uuid_2, uuid_3, uuid_4, uuid_5, uuid_6, uuid_7, uuid_8, uuid_9]
+
+        default_pool = util.default_pool(self.cook_url)
+        active_pools, _ = util.active_pools(self.cook_url)
+        extra_pool = None
+
+        # If using pools, submit jobs to another pool
+        if len(active_pools) > 0:
+            for pool in active_pools:
+                if pool['name'] != default_pool:
+                    extra_pool = pool['name']
+                    cp, uuids = cli.submit(command, self.cook_url, submit_flags=f'--cpus 0.2 --mem 32 --pool {extra_pool}')
+                    self.assertEqual(0, cp.returncode, cp.stderr)
+                    uuid_10 = uuids[0]
+                    all_uuids.append(uuid_10)
+                    cp, uuids = cli.submit(command, self.cook_url, submit_flags=f'--cpus 0.2 --mem 32 --pool {extra_pool}')
+                    self.assertEqual(0, cp.returncode, cp.stderr)
+                    uuid_11 = uuids[0]
+                    all_uuids.append(uuid_11)
+                    break
```

Comment:
Why `for` then `break`? Couldn't we just do `extra_pool = next(p for p in active_pools if p['name'] != default_pool)`